src/data_store/data_store_multi_images.cpp

```diff
@@ -26,20 +26,17 @@
 ////////////////////////////////////////////////////////////////////////////////
 
 #include "lbann/data_store/data_store_multi_images.hpp"
-#include "lbann/data_readers/data_reader_triplet.hpp"
 #include "lbann/data_readers/data_reader_multi_images.hpp"
 #include "lbann/utils/exception.hpp"
 #include "lbann/utils/options.hpp"
 #include "lbann/utils/timer.hpp"
-#include <sys/stat.h>
 
 namespace lbann {
 
-std::unordered_map<std::string, size_t> name_2_idx;
 std::unordered_map<size_t, std::string> idx_2_name;
```

Comment:
name_2_idx and idx_2_name do not seem to be in use anymore. If this is the case, please remove them.